pkg/crane/crane_test.go

```diff
@@ -195,6 +194,20 @@ func TestCraneRegistry(t *testing.T) {
 	if err := compare.Images(dstPulled, copied); err != nil {
 		t.Fatal(err)
 	}
+
+	// List Catalog
+	repos, err := crane.Catalog(u.Host)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(repos) != 2 {
+		t.Fatalf("wanted 2 repos, got %d", len(repos))
+	}
+
+	_, err = crane.Catalog("registry.honk")
```

Comment:
I believe this will actually hit the internet, which I'd like to avoid. This might be covered by [these tests](https://github.com/google/go-containerregistry/blob/60c7f4d7cd5b5c98fc8113183a91a5b8b374170a/pkg/crane/crane_test.go#L420-L421) already?

Reply:
removed, PTAL